docs/User/Frequently_asked_questions.md

```diff
@@ -139,9 +161,11 @@ For configuring the license server API port, see *AutoPass License Server User G
 
 ## <a name="SECTION_HH5_KNP_NSB"/> What is the IP address used in the run scripts?
 
-By default, Swarm Learning framework uses the host network to connect and communicate with its peers. In this case, the IP addresses represent the IP addresses or FQDN of the host systems on which the containers are run.
+The `--host-ip` and `slhostip` IP addresses in the run scripts and the SWOP profile are the IP addresses of the host machine, where the respective containers are running on the host machine. Based on access, user can even use the FQDN of the host system.
```

Comment:
Replace this answer using the new content of "docs/Install/IP_address_of_host_systems.md"